docs/source/migrate-from-dynamodb.rst

```diff
@@ -0,0 +1,217 @@
+=====================
+Migrate from DynamoDB
+=====================
+
+This page explains how to fill the ``source`` and ``target`` properties of the `configuration file </configuration>`_ to migrate data:
+
+- from a DynamoDB table, a ScyllaDB Alternator table, or a `DynamoDB S3 export <https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/S3DataExport.HowItWorks.html>`_,
+- to a DynamoDB table or a ScyllaDB Alternator table.
+
+In file ``config.yaml``, make sure to keep only one ``source`` property and one ``target`` property, and configure them as explained in the following subsections according to your case.
+
+----------------------
+Configuring the Source
+----------------------
+
+The data ``source`` can be a DynamoDB or Alternator table, or a DynamoDB S3 export.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Reading from DynamoDB or Alternator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In both cases, when reading from DynamoDB or Alternator, the type of source should be ``dynamodb`` in the configuration file. Here is a minimal ``source`` configuration to read a DynamoDB table:
+
+.. code-block:: yaml
+
+  source:
+    type: dynamodb
+    table: <table>
+    region: <region>
+
+Where ``<table>`` is the name of the table to read, and ``<region>`` is the AWS region where the DynamoDB instance is located.
+
+To read from the Alternator, you need to provide an ``endpoint`` instead of a ``region``:
+
+.. code-block:: yaml
+
+  source:
+    type: dynamodb
+    table: <table>
+    endpoint:
+      host: http://<host>
+      port: <port>
+
+Where ``<host>`` and ``<port>`` should be replaced with the host name and TCP port of your Alternator instance.
+
+In practice, your source database (DynamoDB or Alternator) may require authentication. You can provide the AWS credentials with the ``credentials`` property:
+
+.. code-block:: yaml
+
+  source:
+    type: dynamodb
+    table: <table>
+    region: <region>
+    credentials:
+      accessKey: <access-key>
+      secretKey: <secret-key>
+
+Where ``<access-key>`` and ``<secret-key>`` should be replaced with your actual AWS access key and secret key.
+
+The Migrator also supports advanced AWS authentication options such as using `AssumeRole <https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html>`_. Please read the `configuration reference </configuration#aws-authentication>`_ for more details.
```

Comment:
Couldn't get the "configuration reference" link to work

Reply:
Ah, good catch. It was working locally but it does not work on GitHub pages because the website URL now contains a prefix “/scylla-migrator”. I will fix this.